08-components-lab/src/03-colorpicker.jsx

```diff
@@ -1,17 +1,61 @@
-/**
- * Lab 03:
- *
- * Implement a color picker component.
- *
- * Component should show an <input> field to select a color, and below a <div>
- * When a color is selected, that color should be used as the background color
- * of the <div>
- *
- * Bonus: Allow external code to pass some text to be used as the contents of the
- * colored <div>
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+/**
+ * Lab 03:
+ *
+ * Implement a color picker component.
+ *
+ * Component should show an <input> field to select a color, and below a <div>
+ * When a color is selected, that color should be used as the background color
+ * of the <div>
+ *
+ * Bonus: Allow external code to pass some text to be used as the contents of the
+ * colored <div>
+ */
+
+import React from 'react';
+
+export default class ColorPicker extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            colors: ["white", "black", "blue", "red", "green"],
+            pickedColor: "white",
+            input: "",
+        };
+    }
+
+    changeColor = (ev) => {
+        console.log(this.state.input);

```

Comment:
You can use same es6 function arguments destructuring as I explained in previous excercises